### ISSUE TITLE
Crash

### ISSUE THREAD

**Issue by gdonisi** [Tobi823/ffupdater#306]
The app crashes when I click on a yellow warning

Stacktrace:
```
java.lang.IllegalArgumentException: Required value was null.
	at de.marmaro.krt.ffupdater.dialog.AppWarningDialog$Companion.newInstance(Unknown Source:46)
	at de.marmaro.krt.ffupdater.AddAppActivity$AvailableAppsAdapter.onBindViewHolderApp$lambda-1(Unknown Source:12)
	at de.marmaro.krt.ffupdater.AddAppActivity$AvailableAppsAdapter.c(Unknown Source:0)
	at de.marmaro.krt.ffupdater.a.onClick(Unknown Source:10)
	at android.view.View.performClick(View.java:7455)
	at android.view.View.performClickInternal(View.java:7432)
	at android.view.View.access$3700(View.java:835)
	at android.view.View$PerformClick.run(View.java:28810)
	at android.os.Handler.handleCallback(Handler.java:938)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loopOnce(Looper.java:201)
	at android.os.Looper.loop(Looper.java:288)
	at android.app.ActivityThread.main(ActivityThread.java:7870)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:550)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1003)
Timestamp: 2023-01-05T11:31:58.782

```
Device information:
| Key | Value |
| --- | --- |
| FFUpdater version | 77.6.1 (123) release |
| Device | Mi 9T (arrow_davinci, davinci, sm6150) |
| Manufacturer | Xiaomi (Xiaomi) | 
| Supported ABIs | arm64-v8a, armeabi-v7a, armeabi |
| Android version | 12 (SDK: 32) |
| OS | release, arrow, release-keys, 1668482416000 |

https://user-images.githubusercontent.com/82442402/210761213-27e95d25-4277-4c4c-873f-f91895c3f6d3.mp4

**Comment by Tobi823** [comment]
This is an embarrassing bug 🙈  

The real problem is that the warn icon is only displayed for apps which have no warnings. For example Firefox Focus beta should have a warning and not Firefox Focus.

I will release the fix today.

**Comment by Tobi823** [comment]
And thanks for the screen recording - this helped a lot :)

**Comment by Tobi823** [comment]
Fix version 77.6.2 is now released.
